applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian.hpp

```diff
@@ -269,7 +269,7 @@ class UpdatedLagrangian
     /**
      * Called at the beginning of each solution step
      */
-    void InitializeSolutionStep(const ProcessInfo& rCurrentProcessInfo) override;
+    virtual void InitializeSolutionStep(const ProcessInfo& rCurrentProcessInfo) override;
```

Comment:
```suggestion
    void InitializeSolutionStep(const ProcessInfo& rCurrentProcessInfo) override;
```

this is only done in the baseclass